apps/site/src/app/(private)/dashboard/(multi)/my-profile/page.tsx

```diff
@@ -1,6 +1,6 @@
 import { redirect } from "next/navigation"
-import ProfilePageBody from "./PageBody"
 import { auth } from "@/context/auth"
+import { MyProfile } from "@padma/metajob-ui"
```

Comment:
### Issue: Hardcoded Import

 **Description**
We cannot import like this: `import { MyProfile } from "@padma/metajob-ui"`.  
We need to ensure that any theme can be used. Avoid hardcoding `@padma/metajob-ui`.